redirect_circleci_artifacts.yml keeps failing

I keep receiving notifications that https://github.com/bids-standard/bids-specification/blob/master/.github/workflows/redirect_circleci_artifacts.yml is failing, and indeed looking at https://github.com/bids-standard/bids-specification/actions/workflows/redirect_circleci_artifacts.yml shows them all red and sample output is

```
2021-11-29T11:16:45.1868792Z ##[group]Run curl --fail  | grep $GITHUB_SHA
2021-11-29T11:16:45.1869624Z [36;1mcurl --fail  | grep $GITHUB_SHA[0m
2021-11-29T11:16:45.1915837Z shell: /usr/bin/bash -e {0}
2021-11-29T11:16:45.1916431Z ##[endgroup]
2021-11-29T11:16:45.2134994Z curl: no URL specified!
2021-11-29T11:16:45.2140224Z curl: try 'curl --help' or 'curl --manual' for more information
2021-11-29T11:16:45.2160362Z ##[error]Process completed with exit code 1.
2021-11-29T11:16:45.2188732Z Cleaning up orphan processes
```

--- a/.github/workflows/redirect_circleci_artifacts.yml
+++ b/.github/workflows/redirect_circleci_artifacts.yml
@@ -5,13 +5,9 @@ jobs:
     name: Run CircleCI artifacts redirector
     steps:
       - name: GitHub Action step
-        id: step1
         uses: larsoner/circleci-artifacts-redirector-action@master
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           artifact-path: 0/bids-spec.pdf
           circleci-jobs: build_docs_pdf
           job-title: Check the rendered PDF version here!
-      - name: Check the URL
-        run: |
-          curl --fail ${{ steps.step1.outputs.url }} | grep $GITHUB_SHA

